routes/ppe.js

```diff
@@ -64,39 +72,99 @@ router.post('/',
+        // Preview only available when sending through an Ethereal account
+        console.log("Preview URL: %s", nodemailer.getTestMessageUrl(info));
+        // Preview URL: https://ethereal.email/message/WaQKMgKddxQDoou...  
+      });
+
+      console.log("USER: " + user.id)
+      let records = [];
+      for(var i = 0; i <req.body.PPETypeId.length; i++){
+        records.push({
+          name: req.body.name,
+          PPETypeId: req.body.PPETypeId[i],
+          quantity: req.body.quantity[i],
+          email: req.body.email,
+          contact: req.body.contact,
+          latitude: req.body.latitude,
+          longitude: req.body.longitude,
+          ProofId: proof.id,
+          UserId: user.id
+        })
       }
+      
+      request.post(
+        INCOMING_WEBHOOK,
+        {
+          json: {
+            text: `A new ${req.body.mode} was generated`
+          }
+        },
+        (error, res, body) => {
+          if (error) {
+            console.error(error)
+            return
+          }
+          console.log(`statusCode: ${res.statusCode}`)
+          console.log(body)
+        }
+      )
       if (req.body.mode === 'availability') {
-        const availability = await models.Availability.create(record);
-        findMatches(availability, 'Availability', 'onCreate');
-        return res.render('ppe-thanks', { forId: availability.id, forType: 'Availability' });
+        const availability = await models.Availability.bulkCreate(records);
+        // findMatches(availability, 'Availability', 'onCreate');
+        return res.render('ppe-thanks', { forId: availability.id, forType: 'Availability', statusLink: statusLink });
       }
       else if (req.body.mode === 'requirement') {
-        record.canBuy = req.body.canBuy;
-        const requirement = await models.Requirement.create(record);
-        findMatches(requirement, 'Requirement', 'onCreate');
-        return res.render('ppe-thanks', { forId: requirement.id, forType: 'Requirement' });
+        // record.canBuy = req.body.canBuy;
+        for(let r of records){
+          r.canBuy = req.body.canBuy;
+        }
+        const requirement = await models.Requirement.bulkCreate(records);
+        // findMatches(requirement, 'Requirement', 'onCreate');
+        return res.render('ppe-thanks', { forId: requirement.id, forType: 'Requirement', statusLink: statusLink });
       }
       else if (req.body.mode === 'manufacturing') {
-        record.remarks = req.body.remarks;
-        const manufacturing = await models.Manufacturing.create(record);
-        return res.redirect('/ppe/map');
+        // record.remarks = req.body.remarks;
+        for(let r of records){
+          r.remarks = req.body.remarks;
+        }
+        const manufacturing = await models.Manufacturing.bulkCreate(records);
+        return res.render('ppe-thanks', { forId: manufacturing.id, forType: 'Manufacturing', statusLink: statusLink });
```

Comment:
All 3 renders could be refactored out of the conditionals

Reply:
yes right. We had a different render for manufacturing till now, so it seemed a good idea to have them inside separate conditionals, but now we could move them out into one. Good catch